ADAL/src/cache/ADTokenCacheItem+Internal.h

```diff
@@ -27,12 +27,14 @@
 @class ADAuthenticationError;
 @class ADAuthenticationResult;
 
-@interface ADTokenCacheItem (Internal)
+@interface ADTokenCacheItem ()
+
+@property (readwrite) NSMutableDictionary * additionalClient;
+@property (readonly) NSDictionary * additionalServer;
```

Comment:
You can only add properties in an extension, unless you provide some other mechanism for their storage. Adding them like this means they'll get auto-synthesized and I don't have to bother providing implementation for them. :)